Commands/Default.sublime-commands

```diff
@@ -59,5 +59,9 @@
     {
         "caption": "LSP: Rename Symbol",
         "command": "lsp_symbol_rename"
+    },
+    {
+        "caption": "LSP: Execute command",
```

Comment:
Would it be possible for users to register custom commands via JSON configuration? Instead of running "LSP: Execute command" and manually type "build-import" (making sure there are no typos) it would good to additionally support the following workflow

1. register commands in LSP client settings 
```json
{
  "Metals": {
    "commands": [
      { "command": "build-import", "arguments": [] }
    ]
  }
}
```
2. run the "Metals: Import build" command directly without quick panel prompt

Reply:
The example on how to register such commands is in the PR description. Users don't have to type manually "build-import", they have to add it to the client config once. 
Next time they can select the commands that they registered.

Once Metal is stable enough we can add a default client + config to this project, this way users don't event have to configure the client, it comes with the plugin and with a set of predefined commands.

Reply:
I'm sorry I totally missed that part! Some more thoughts

- It would be good to allow a more readable title for the command "Metals: Import build" instead of "build-import".
- (not a big deal) It would be nice if we could avoid the quick panel and allow users to run "Build import" directly as a unique command.
- Note that `workspace/executeCommand` supports `CommandParams.arguments`, we don't use it in Metals but it's part of the spec.

Reply:
make sense. I will add labels for the commands and allow arguments support

Reply:
@ayoub-benali also consider using the more fancy sublime_plugin.ListInputHandler and sublime_plugin.TextInputHandler classes (see plugin/rename.py for an example usage).

Reply:
@rwols and the proper way to test this would be through sublime testing like `test_edit.py` ?

Reply:
@rwols since I moved "class LspExecuteCommand" to it own file `plugin/execute_command.py` the command doesn't appear in the command pallet. Is there some mapping I need to do? `lsp_execute` -> `LspExecuteCommand`

Reply:
editing "boot.py" fixed the issue